### PR TITLE
Export to CSV file depending on the interest of the user

### PR DESCRIPTION
**Pull request by shanmu9898**

### CODE REVIEW

**Comment by shanmu9898** [comment]
Closes #68 #69 #94 #95 #97 #98 #99 #100 #108 #109 #110 #115 #119

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -111,26 +111,26 @@ Forms an English class containing the first, second and fifth contact in the lis
...
-+
-image::appointmentAdded.jpg[width="800"]
-    Figure 3: Appointment added successfully example
-+
+ * `set_appointment t/Tutoring session s/02-04-2018 19:00 e/02-04-2018 20:00` +
+ Sets up an appointment on April 2nd, 2018, from 7pm to 8pm.
+ +
+ image::appointmentAdded.jpg[width="800"]
+     Figure 3: Appointment added successfully example
+ +
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: This change causes some display inconsistency with the Setting up an appointment part, so you should revert it.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -233,14 +233,15 @@ Shows a list of all of the specified `TYPE`. +
 Format: `list TYPE`.
 
 ****
-* `TYPE` can be of the following: `contact`, `student`, `task`, `appointment`.
+* `TYPE` can be of the following: `contact`, `student`, `task`, `appointment`, `shortcut`.
 * `TYPE` cannot be empty.
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: The change you made to the list command now makes type plural, so it should be the following: "contacts", "students", "tasks", "shortcuts", appointments"

Be careful that your command is up to date with the UserGuide, we have UserTesting today!

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -233,14 +233,15 @@ Shows a list of all of the specified `TYPE`. +
...
 ****
 
 Examples:
 
 * List all student: `list student` +
-* List all task: `list task`
+* List all task: `list task` +
+* List all task: `list shortcut`
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: `list students` 
`list tasks`
`list shortcuts`

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -409,50 +410,84 @@ Format: `exit`
 TeachConnect data are saved in the hard disk automatically [even while sharing TeachConnect] after any command that changes the data. +
 There is no need to save manually.
 
-=== Importing the contacts : `import` `[since v1.2]`
+=== Importing the contacts : `import` `[since v1.4]`
 
 TeachConnect allows you to import contacts from a different TeachConnect file by specifying the location of the file. +
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: TeachConnect allows you...
-->
Imports contacts from a different....
Consistency with other commands

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -409,50 +410,84 @@ Format: `exit`
...
 
 Examples:
-Importing Contacts : `import ./data/importsample.xml`
+* Importing Contacts : `import ./data/importsample.xml`
 
-=== Exporting the contacts : `export` `[since v1.2]`
+=== Exporting the contacts : `export` `[since v1.4]`
 
-TeachConnect allows you to export contacts from your TeachConnect by specifying the name of the file you want to save it in and the path where you want to save it. It can export the contacts based on a given range of indexes or a given tag or a given tag in a range of indexes.
+TeachConnect allows you to export contacts from your TeachConnect by specifying the name of the file you want to save it in and the path where you want to save it. It can export the contacts based on a given range of indexes or a given tag or a given tag in a range of indexes. By specifying the type of the export you want it saves either only the xml file or both the xml and Csv file.
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: TeachConnect allows you....
-->
Exports contacts...
TeachConnect can export

--> consistency with other commands

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -409,50 +410,84 @@ Format: `exit`
...
 Examples:
 
-Exporting contacts : `export n/StudentsFile1 r/all t/students p/./data`
-Exporting contacts : `export n/StudentsFile2 r/1,10 t/students p/./data`
+* Exporting contacts : `export n/StudentsFile1 r/all t/students p/./data te/normal`
+* Exporting contacts : `export n/StudentsFile2 r/1,10 t/students p/./data te/excel`
+
+=== Adding your own shortcut : `shortcut` `[since v1.4]`
+
+TeachConnect also allows you to set your own personal shortcut for any of the commands above.
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: Similarly:
`Sets your own shortcut...`

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -409,50 +410,84 @@ Format: `exit`
...
-=== Importing the contacts : `import` `[since v1.2]`
+=== Importing the contacts : `import` `[since v1.4]`
 
 TeachConnect allows you to import contacts from a different TeachConnect file by specifying the location of the file. +
 
 Format: `import [TYPE] pathname`
 
 Examples:
-Importing Contacts : `import ./data/importsample.xml`
+* Importing Contacts : `import ./data/importsample.xml`
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: Wrong formatting. See the preview to see the error.

Also if you want to improve the consistency for examples: it should be in the format:
`import....`
Imports contacts from importsample.xml

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -409,50 +410,84 @@ Format: `exit`
...
+You cannot set the shortcut word to a already preregistered command. +
+Your shortcut word cannot be more than a single word.
+
+Examples:
+* Personalised Alias : `shortcut list l`
+* Personalised Alias : `shortcut add a`
+
+=== Deleting the shortcut you created : `delete_shortcut` `[since v1.4]`
+
+TeachConnect allows you to delete your personalised Alias if you don't want them or if you created them by mistake.
```
Comment (Sisyphus25) [inline docs/UserGuide.adoc]: Deletes your personalised Alias....

**Pull request merged**